packages/block-editor/src/components/index.js

```diff
@@ -89,7 +89,11 @@ export { default as withColorContext } from './color-palette/with-color-context'
 
 export { default as __unstableBlockSettingsMenuFirstItem } from './block-settings-menu/block-settings-menu-first-item';
 export { default as __unstableInserterMenuExtension } from './inserter-menu-extension';
-export { default as __experimentalPreviewOptions } from './preview-options';
+export {
```

Comment:
Can you explain why `__experimentalPreviewOptions` and `coreDeviceTypes` are proposed to become part of public API? They don't seem to be used outside of the package where they are defined.